File: VS2022/CS/Forms/CSCal2NumForms/CSCal2NumForms/Form1.cs
namespace CSCal2NumForms
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            double a = Convert.ToDouble(tB1.Text);
            double b = Convert.ToDouble(tB2.Text);
            tB3.Text = (a + b).ToString();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            double a = Convert.ToDouble(tB1.Text);
            double b = Convert.ToDouble(tB2.Text);
            tB3.Text = (a - b).ToString();
        }

        private void button3_Click(object sender, EventArgs e)
        {
            double a = Convert.ToDouble(tB1.Text);
            double b = Convert.ToDouble(tB2.Text);
            tB3.Text = (a * b).ToString();
        }

        private void button4_Click(object sender, EventArgs e)
        {
            double a = Convert.ToDouble(tB1.Text);
            double b = Convert.ToDouble(tB2.Text);
            tB3.Text = (a / b).ToString();
        }
    }
}

File: VS2022/VB/Form/VBCal2NumForms/VBCal2NumForms/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Dim FN, SN, Ans As Double
        FN = tB1.Text
        SN = tB2.Text
        Ans = FN + SN
        tB3.Text = Ans
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Dim FN, SN, Ans As Double
        FN = tB1.Text
        SN = tB2.Text
        Ans = FN - SN
        tB3.Text = Ans
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        Dim FN, SN, Ans As Double
        FN = tB1.Text
        SN = tB2.Text
        Ans = FN * SN
        tB3.Text = Ans
    End Sub

    Private Sub Button4_Click(sender As Object, e As EventArgs) Handles Button4.Click
        Dim FN, SN, Ans As Double
        FN = tB1.Text
        SN = tB2.Text
        Ans = FN / SN
        tB3.Text = Ans
    End Sub

End Class
